tests/integration/src/setup.rs

```diff
@@ -175,6 +176,8 @@ impl ExtBuilder {
 
         let mut ext = sp_io::TestExternalities::new(t);
         ext.execute_with(|| System::set_block_number(1));
+        #[cfg(feature = "shibuya")]
```

Comment:
IMHO, you shouldn't do this here - let the tests that needs it set it up.

Tests will be more clear since this "setup" action will be clearly visible in the test code.

Reply:
Since most of XVM tests required this, it made more sense to put it here.
But I'll move it to individual tests

Reply:
It's not a generic thing so it's better for tests to configure it themselves.